src/locale/misc/formDialog/en.yml

```diff
@@ -1,26 +1,27 @@
-createNew: 
-    heading: Create new {resource}
-    campaign:
-        name: Name
-        description: Description
-        visibility:
-            heading: Visibility
-            private: Private
-            public: Public
-        status:
-            heading: Status
-            published: Published
-            draft: Draft
-    event:
-        campaign: Campaign
-        title: Title
-        link: Link
-        start: Start Time
-        end: End Time
-        activity: Activity
-        place: Place
-        info: Information on signup
-        minNo: Mininum participants
+create: Create new {resource} 
+edit: Edit {resource} 
```

Comment:
It's better to move these into `campaign` and `event` instead of passing in variables. In English this is fine, but the verb might be different for campaigns and events, and other nouns depending on things like grammatical gender or just semantic context, so reusing part of the string won't work.